pkg/operator/ceph/file/filesystem.go

```diff
@@ -84,8 +84,10 @@ func createFilesystem(
 
 	// set the number of active mds instances
 	if fs.Spec.MetadataServer.ActiveCount > 1 {
-		if err = client.SetNumMDSRanks(context, fs.Namespace, fs.Name, fs.Spec.MetadataServer.ActiveCount); err != nil {
-			logger.Warningf("failed setting active mds count to %d. %+v", fs.Spec.MetadataServer.ActiveCount, err)
+		if err = client.SetNumMDSRanks(context, cephVersion.Name, fs.Namespace, fs.Name,
```

Comment:
Go doesn't set hard line limits. While super-long-lines are hard to read and should be split, lines around 100 characters long shouldn't be an issue with modern editors. Splitting these lines adds the appearance of complexity to the conditional block where there isn't any. For standard error checking blocks, the convention I have seen is to keep the condition and the error message each on one line unless they are very much longer than 100 chars.

Also, these line-length changes aren't really in-scope with the other changes, and if you still feel strongly about them, they should be submitted in a separate PR.